en/docs/develop-components/bring-your-own-image.md

```diff
@@ -4,7 +4,7 @@ Deploy and manage (prebuilt) container images from external container registries
 
 !!! info
     - This feature is currently only available on [Private Data Planes (PDP)](https://wso2.com/choreo/docs/choreo-concepts/data-planes/#private-data-plane).
-    - This feature currently only supports the following component types: service components, web applications, scheduled triggers, and manual triggers.
+    - This feature currently only supports the following component types: service components, web applications, scheduled tasks, and manual tasks.
```

Comment:
Shall we merge the two points in a manner similar to the following:
This feature is currently available only on [Private Data Planes (PDP)](https://wso2.com/choreo/docs/choreo-concepts/data-planes/#private-data-plane) and supports the following component types:
 - Services
 - Web applications
 - Scheduled tasks
 - Manual tasks